kit/ChildSession.cpp

```diff
@@ -631,7 +631,7 @@ bool ChildSession::loadDocument(const char * /*buffer*/, int /*length*/, const S
     const bool loaded = _docManager->onLoad(getId(), getJailedFilePathAnonym(), renderOpts);
     if (!loaded || _viewId < 0)
     {
-        LOG_ERR("Failed to get LoKitDocument instance for [" << getJailedFilePathAnonym() << "].");
+        LOG_FTL("Failed to get LoKitDocument instance for [" << getJailedFilePathAnonym() << "].");
```

Comment:
Loading can fail for many reasons. It's an error, but not fatal. We can load other documents and even the same document again. FTL signals unrecoverable error, where we exit the process because we cannot continue running. This is nothing like that. ERR is reasonable here.